coalib/bears/Bear.py

```diff
@@ -249,8 +249,14 @@ def execute(self, *args, **kwargs):
             result = self.run_bear_from_section(args, kwargs)
             return [] if result is None else list(result)
         except (Exception, SystemExit):
-            self.warn('Bear {} failed to run. Take a look at debug messages'
-                      ' (`-V`) for further information.'.format(name))
+            if self.kind():
```

Comment:
However, I have replaced it with the condition `bear.kind() == BEAR_KIND.GLOBAL`. Is that what you implied to say ?